readme.md

```diff
@@ -49,7 +62,24 @@ Worker objects are extended [browser objects](#browser-objects) which contain th
 * `status`: A string representing the current status of the worker.
   * Possible statuses: `"running"`, `"queue"`.
 
-### BrowserStack.createClient( settings )
+#### screenshot objects
+
+A "screenshot object" is a configuration object to run a screenshot job.
+
+It is a plain object with properties defined on [BrowserStack's Screenshot API reference](https://www.browserstack.com/screenshots/api#generate-screenshots).
```

Comment:
The BrowserStack docs for this are actually really bad. I'm going to inline the properties.